frontend/ui/widget/textwidget.lua

```diff
@@ -41,9 +41,9 @@ function TextWidget:updateSize()
     if not tsize then
         self._length = 0
     else
-        self._length = tsize.x
+        self._length = math.ceil(tsize.x)
```

Comment:
Moreover, the problem i witnessed had more to do with the `self._height = math.ceil(self.face.size * 1.5)`, it's this `*1.5` that was giving non-integer heights (face.size = 7 => height = 10.5). RenderTextsizeUtf8 should always give integer I guess.
I ceil()'ed both width and height for consistency and assurance.